src/config/dynamic-theme-fixes.config

```diff
@@ -4595,6 +4595,15 @@ INVERT
 
 ================================
 
+wikidata.org
+
+CSS
+.wd-mp-headerimage {
```

Comment:
```
wikidata.org

INVERT
.wd-mp-headerimage

================================

```
We prefer this so it will have a `hue-rotate(180deg)` So colors will be kind of 'originial'

Reply:
Thanks, changed it